modules/rules_android_ndk/0.1.2/presubmit.yml

```diff
@@ -0,0 +1,37 @@
+bcr_test_module:
+  module_path: examples/basic
+  matrix:
+    bazel: ["7.2.1", "rolling"]
+  tasks:
+    run_test_module_ubuntu:
+      name: "Verify build targets with bzlmod"
+      platform: "ubuntu2004"
+      bazel: ${{ bazel }}
+      environment:
+        ANDROID_NDK_HOME: /opt/android-ndk-r25b
+      build_flags:
+        - "--android_platforms=//:arm64-v8a"
+      build_targets:
+        - "//java/com/app:app"
+    run_test_module_macos:
+      name: "Verify build targets with bzlmod on MacOS"
+      platform: "macos"
+      bazel: ${{ bazel }}
+      environment:
+        ANDROID_NDK_HOME: "/Users/buildkite/android-ndk-r25b"
+      build_flags:
+        - "--android_platforms=//:arm64-v8a"
+      build_targets:
+        - "//java/com/app:app"
+    run_test_module_windows:
+      name: "Verify build targets with bzlmod on Windows"
+      platform: "windows"
+      bazel: ${{ bazel }}
+      environment:
+        ANDROID_NDK_HOME: "c:/android_ndk/r25b"
+        BAZELCI_LOCAL_RUN: true
```

Comment:
This is a workaround for a known issue with remote caching + Windows + Android NDK. We're working on validating a fix internally.